frontend/src/pages/journal/components/journal-sidebar/journal-sidebar.tsx

```diff
@@ -29,7 +32,11 @@ const JournalSidebar: React.FC<Properties> = ({
   setIsSidebarShown,
 }) => {
   const dispatch = useAppDispatch();
-
+  const [deletedChat, setDeleteChat] = useState<null | number>(null);
```

Comment:
Please check comments of @OlenaYatseniuk's PR

Reply:
done